qiskit_ibm_runtime/utils/noise_learner_result.py

```diff
@@ -14,47 +14,117 @@
 
 from __future__ import annotations
 
-from dataclasses import dataclass
 from typing import Any, Iterator, Sequence
 
 from qiskit.circuit import QuantumCircuit
 from qiskit.quantum_info import PauliList
 
 
-@dataclass(frozen=True)
-class NoiseLearnerDatum:
-    """A container for a single noise learner datum.
+class PauliLindbladError:
+    """
+    A container for the generators and rates of an error channel in the sparse Pauli-Lindblad noise
+    model.
```

Comment:
I like this! A bunch of methods of the AER one (like the `to_pauli_error`) seems like a nice to have, it will be nice to replace the two classes with one. The surviving one should probably live in qiskit